fpmsyncd/routesync.cpp

```diff
@@ -788,6 +831,8 @@ void RouteSync::onLabelRouteMsg(int nlmsg_type, struct nl_object *obj)
     struct nl_addr *daddr;
     char destaddr[MAX_ADDR_SIZE + 1] = {0};
 
+    sendOffloadReply(route_obj);
```

Comment:
Hope you have a test case to test this MPLS label route.

Reply:
I don't add additional flows for MPLS route handling. So it should be covered by existing MPLS route tests.